model.ipynb

```diff
@@ -0,0 +1,433 @@
+{
```

Comment:
<p>Typically comments go under the function definition!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/loglabs/ttb/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Done. Moved